### ISSUE TITLE
Cannot understand week 8 lecture brief question 4 Person class

### ISSUE THREAD

**Issue by tayxuenye** [nus-cs2103-AY2425S1/forum#509]
![image](https://github.com/user-attachments/assets/0084d25f-de19-4e4b-a4ce-ca00f3e3d07e)
Hi, this question is from week 8's brief. I don't really understand the Person class. How can a Person class contain both a parent and a child? Shouldn't a person be either a parent or a child? Not very sure what the bi-directional arrow in option C (the correct ans) means too.

**Comment by Feng1231** [comment]
The attributes can be viewed as a 'has a' relationship with the instance.
In simpler terms, it means, A person can have a parent, and a person have a child.
Suppose Person A has a child, Person B.
Then A.child = B.
When you look at Person B,
Person B definitely has a parent, which is Person A
B.parent = A.

This is shown by the bi directional arrow in option C,
since both object stores a reference to each other, hence they can navigate to each other

**Comment by tayxuenye** [comment]
Thank you

**Comment by damithc** [comment]
Thanks for the explanation, @Feng1231 

> How can a Person class contain both a parent and a child?

@tayxuenye An association line should be interpreted as 'associated with', not as 'contains'. 
The way to interpret (c) is,
`Person` objects can be _associated with_ other `Person` objects, in which object at one end is playing the role of `parent` while the object at the other end is playing the role of `child`.

**Comment by tayxuenye** [comment]
Thank you prof that's clearer!
